baseballAPI/ApiModels/TeamStatsMapper.cs

```diff
@@ -0,0 +1,212 @@
+﻿using BaseballAPI.RepositoryModels;
+
+namespace BaseballAPI.ApiModels
+{
+    public class TeamStatsMapper : ITeamStatsMapper
+    {
+        public TeamStatsMapper()
+        {
+
+        }
+
+        public TeamStats Map(Teams teams)
+        {
+            var teamsStats = new TeamStats()
+            {
+                YearId = teams.YearId,
+                LgId = teams.LgId,
+                TeamId = teams.TeamId,
+                FranchId = teams.FranchId,
+                DivId = teams.DivId,
+                DivWin = teams.DivWin,
+                Wcwin = teams.DivWin,
+                LgWin = teams.LgWin,
+                Wswin = teams.Wswin,
+                Name = teams.Name,
+                Park = teams.Park,
+                TeamIdbr = teams.TeamId,
+                TeamIdlahman45 = teams.TeamIdlahman45,
+                TeamIdretro = teams.TeamIdretro
+            };
+
+            ConvertOptionalParamsToNonNullable(teams, teamsStats);
+
+            return teamsStats;
+        }
+
+        private void ConvertOptionalParamsToNonNullable(Teams teams, TeamStats teamsStats)
+        {
+            if (teams.Rank != null)
```

Comment:
c# has a "null-coalescing" operator that could help here: `??`
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-coalescing-operator

I think this could be simplified as `teamsStats.Rank = teams.Rank ?? 0;`

You could also pull out helper function if you think it would be worth it.
Something like:
```
teamsStats.Rank = GetOrZero(teams.Rank);
...
private short GetOrZero(short? value) {
    return value ?? 0;
}
```